src/Network/Network/help/New-AzVirtualNetworkSubnetConfig.md

```diff
@@ -109,6 +119,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -InputObject
+Specifies the nat gateway associated with the subnet configuration
+
+```yaml
+Type: Microsoft.Azure.Commands.Network.Models.PSNatGateway
+Parameter Sets: SetByResource
+Aliases:
```

Comment:
The aliases are not being added. Are you sure that you built the latest changes and ran the update command using these latest changes? If still the aliases are not being added try to completely delete the help file before regenerating.